Bookings/Booking.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Airport_Ticket_Booking_System.Flights;
+using Airport_Ticket_Booking_System.Users;
+
+namespace Airport_Ticket_Booking_System.Bookings;
+
+public class Booking
+{
+    public int Id { get; set; }
+    public Flight Flight { get; set; }
+    public User User { get; set; }
+    public DateTime BookingDate { get; set; } = DateTime.Now;
+    public BookingStatus Status { get; set; } = BookingStatus.Pending;
+
+    public static string ToCsv(Booking booking)
```

Comment:
```suggestion
    public static string ToCsvFormat(Booking booking)
```

To reflect the actual expected result.